### PR TITLE
WIP: added travis.yml

### PR DESCRIPTION
**Pull request by robinlehrmann**
depends on #32. The other PR should be merged at first. Then I will refactor this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # IOTA-PHP
 
+[![Build Status](https://travis-ci.org/Techworker/iota-php.svg?branch=master)](https://travis-ci.org/Techworker/iota-php)
```
Comment (Techworker) [inline README.md:3]: Runs under my username? Are you sure?

**Review by Techworker — APPROVED** [review]
Seems fine apart from the `Techworker` namespace on travis. Thx

File: README.md (hunk 1 above)
Comment (robinlehrmann) [reply README.md:3]: No, do you have the correct url ? :)

Comment (peter279k) [reply README.md:3]: It looks like using this url: `httpS://travis.org//iota-php/client`.
And it has to make sure this repository set the web hook of Travis CI correctly.

**Comment by peter279k** [comment]
Can anyone concern this PR?

**Comment by Techworker** [comment]
I just merge it now, fixes need to be applied on top of that what will go into the master.

**Pull request merged**